src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/spg/SpgExtraordinaryRefusalReceivedProcessor.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.ons.census.fwmt.outcomeservice.converter.spg;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.census.fwmt.common.error.GatewayException;
+import uk.gov.ons.census.fwmt.events.component.GatewayEventManager;
+import uk.gov.ons.census.fwmt.outcomeservice.config.GatewayOutcomeQueueConfig;
+import uk.gov.ons.census.fwmt.outcomeservice.converter.SpgOutcomeServiceProcessor;
+import uk.gov.ons.census.fwmt.outcomeservice.dto.SpgOutcomeSuperSetDto;
+import uk.gov.ons.census.fwmt.outcomeservice.message.GatewayOutcomeProducer;
+import uk.gov.ons.census.fwmt.outcomeservice.template.TemplateCreator;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static uk.gov.ons.census.fwmt.outcomeservice.config.GatewayEventsConfig.CESPG_OUTCOME_SENT;
+import static uk.gov.ons.census.fwmt.outcomeservice.enums.EventType.REFUSAL_RECEIVED;
+import static uk.gov.ons.census.fwmt.outcomeservice.enums.SurveyType.spg;
+
+@Component("EXTRAORDINARY_REFUSAL_RECEIVED")
+public class SpgExtraordinaryRefusalReceivedProcessor implements SpgOutcomeServiceProcessor {
+
+  @Autowired
+  private GatewayOutcomeProducer gatewayOutcomeProducer;
+
+  @Autowired
+  private GatewayEventManager gatewayEventManager;
+
+  @Override
+  public UUID process(SpgOutcomeSuperSetDto outcome, UUID caseIdHolder) throws GatewayException {
+    UUID caseId = (outcome.getCaseId() != null) ? outcome.getCaseId() : caseIdHolder;
+    String eventDateTime = outcome.getEventDate().toString();
```

Comment:
Use SimpleDateFormatter to convert to correct format YYYY-MM-DD THH:MM:SS.ssssssZ

Reply:
Done.